DATS.json

```diff
@@ -0,0 +1,203 @@
+  },
+  "dates": [
+    {
+      "@id": "https://w3id.org/dats/schema/date_info_schema.json",
+      "@type": "Date",
+      "date": "2020-05-04",
+      "type": {
+        "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+        "@type": "Annotation",
+        "value": "Start Date"
+      }
+    }
+  ],
+  "creators": [
+    {
+      "name": "The Courtois Project on Neuronal Modelling"
+    }
+  ],
+  "types": [
+    {
+      "@id": "https://w3id.org/dats/schema/data_type_schema.json",
+      "@type": "DataType",
+      "information": {
+        "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+        "@type": "Annotation",
+        "value": "MRI"
+      }
+    },
+    {
+      "@id": "https://w3id.org/dats/schema/data_type_schema.json",
+      "@type": "DataType",
+      "information": {
+        "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+        "@type": "Annotation",
+        "value": "Basic Demographics"
+      }
+    },
+    {
+      "@id": "https://w3id.org/dats/schema/data_type_schema.json",
+      "@type": "DataType",
+      "information": {
+        "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+        "@type": "Annotation",
+        "value": "BIDS"
+      }
+    }
+  ],
+  "version": "1.1",
+  "availability": "available",
+  "privacy": "Private",
+  "distributions": [
+    {
+    "@id": "http://json-schema.org/draft-04/schema",
+    "@type": "DatasetDistribution",
+    "formats": [
+      "BIDS",
+      "NIfTI",
+      "TSV",
+      "JSON"
+    ],
+    "size" : 1000,
+    "unit" : {
+      "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+      "@type": "Annotation",
+      "value": "GB"
+    },
+    "access": {
+      "landingPage": "https://www.cneuromod.ca/carousel/acquisitions",
+      "authorizations": [
+        {
+          "value": "private"
+        }
+      ]
+    }
+  }
+  ],
+  "isAbout": [
+    {
+      "@id": "https://w3id.org/dats/schema/biological_entity_schema.json",
+      "@type": "BiologicalEntity",
+      "name": "Adult humans"
+    }
+  ],
+  "spatialCoverage": [
+    {
+      "@id": "https://w3id.org/dats/schema/place_schema.json",
+      "@type": "Place",
+      "name": "Québec (province)",
+      "description": "Province of Québec, Canada"
+    }
+  ],
+  "aggregation": "instance of dataset",
+  "dimensions": [
+    {
+      "@id": "https://w3id.org/dats/schema/dimension_schema.json",
+      "@type": "Dimension",
+      "name" : {
+          "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+          "@type": "Annotation",
+          "value": "Structural MRI"
+      },
+      "description": "T1W, T2W, DWI"
+    },
+    {
+      "@id": "https://w3id.org/dats/schema/dimension_schema.json",
+      "@type": "Dimension",
+      "name" : {
+          "@id": "https://w3id.org/dats/schema/annotation_schema.json",
+          "@type": "Annotation",
+          "value": "Functional MRI"
+      },
+      "description": "resting BOLD"
```

Comment:
Most of the functional acquisition are task-based.

Reply:
so `resting BOLD` and `tasks BOLD`?